src/server/plugins/feed/index.js

```diff
@@ -28,8 +28,22 @@ type Socket = {
   on: (event: string, cb: (d?: any) => void) => void
 }
 
+type FetchOptions = {
+  from: Date,
+  to: Date,
+  published: Boolean,
+  viewed: Boolean
+}
+
 const logger: Logger = debug('server:plugins:feed')
 
+const fetchOptions: FetchOptions = {
+  from: '2016-09-17T17:26:17+00:00',
+  to: '2016-09-20T17:26:17+00:00',
```

Comment:
These are hardcoded right now. We'll have to modify `server/orm/events.js` #fetch method to allow undefined `from` and `to` properties. So we can just fetch all for the feed view.